pyomo/solvers/plugins/solvers/CPLEX.py

```diff
@@ -508,6 +508,19 @@ def process_logfile(self):
         self._best_bound = None
         self._gap = None
 
+        # use regular expressions to use multi-line match patterns:
+        root_node_processing_time = re.search(
+            r'Root node processing.*:\n\s+Real time\s+=\s+(\d+\.\d+) sec', output
+        )
+        if root_node_processing_time:
+            results.solver.root_node_processing_time = float(root_node_processing_time.group(1))
+
+        tree_processing_time = re.search(
+            r'(Parallel|Sequential).*\n\s+Real time\s+=\s+(\d+\.\d+) sec', output
```

Comment:
Minor: You can do `(?:Parallel|Sequential)` to make this a non-capturing group. That way we always return 1 group and can do `.group(1)` everywhere.

Reply:
done